markdown/1.x/en/documentation/currency-guide.md

```diff
@@ -105,10 +101,8 @@ Specify the maximum number of digits for all prices.
 `<ff-communication currency-max-digits="4">`
 
 ---
-#### **add-unit-to-filter-groups (Boolean)**
-If present, the unit formatting takes place for all filter groups and it's filters. Note that this is a `Boolean` attribute and is therefore ignoring its value. Read more about [attributes](todo) 
-
-**Example Usage**
-`<ff-communication add-unit-to-filter-groups>`
+#### **add-unit-to-filter-groups (Boolean) (deprecated since 1.3.1)**
+~~If present, the unit formatting takes place for all filter groups and its filters.~~
+Since 1.2.2 the formatting takes places for all filter groups and its filters internally for every response. Setting the attribute was a noop or could lead to false encoding in case of the currency value differs on client and FACT-Finder site. Since 1.3.1 this attribute is always a noop.
```

Comment:
I am wondering if we should use 'noop' word in official documentation. See [NOP definition](https://en.wikipedia.org/wiki/NOP_(code)). Maybe something like "Since 1.3.1 this attribute is not supported and setting it has no effect.". This comment applies to all 'noop' usages.

Reply:
`has no effect` could be a safer construct.